Lib/messages.rb

```diff
@@ -0,0 +1,42 @@
+require 'pg'
```

Comment:
You mentioned in your self reflection that you felt as though this class had more than one responsibility (inserting into a DB, retrieving from a DB etc.), but I'd argue that it's actually fine.

I'd describe the responsibility of the class as managing Messages. It's sensible to keep all of the behaviour for Messages in one place. The alternative would presumably be to have object responsible for the saving of any type of resource, an object for the retrieval of any type of resource etc, which feels like a strange way of organising things.